src/Microsoft.AspNet.Mvc.Core/ActionDescriptor.cs

```diff
@@ -30,6 +30,11 @@ public ActionDescriptor()
 
         public IList<ParameterDescriptor> Parameters { get; set; }
 
+        /// <summary>
+        /// The set of parameters which are common across actions.
+        /// </summary>
+        public IList<ParameterDescriptor> CommonParameters { get; set; }
```

Comment:
Still suggesting `ControllerProperties` or `BoundProperties` for this. CommonParameters isn't really the truth.